modules/appnexusAstBidAdapter.js

```diff
@@ -36,7 +35,7 @@ function AppnexusAstAdapter() {
   let usersync = false;
 
   /* Prebid executes this function when the page asks to send out bid requests */
-  baseAdapter.callBids = function(bidRequest) {
+  baseAdapter.callBids = function(bidRequest, addBidResponse, done) {
```

Comment:
As long as you're rewriting these... you might want to check out [the format proposed](https://gist.github.com/snapwich/72d266f510bd2f83a6eacfb2bb68112d) by @snapwich 

Other adapter maintainers are going to look to these for inspiration... so setting a good pattern will pay dividends in maintenance.